### ISSUE TITLE
Unrecognized field "customSegmentation" when getting project information

### ISSUE THREAD

**Issue by goaaats** [crowdin/crowdin-cli#379]
**Describe the bug**
When trying to sync localization data with the crowdin CLI in a GitHub action, it fails to get project information due to a serialization error.

**To Reproduce**
1. Configuration file: https://github.com/goatcorp/FFXIVQuickLauncher/blob/master/.github/crowdin.yml
2. See the listed error

**Expected behavior**
The CLI should be able to fetch project information correctly and synchronize localization data.

**Environment:**
 - OS: Ubuntu
 - Version: 20.04
 - CLI: crowdin/cli:3.6.3 (via crowdin/github-action@1.1.1)

**Screenshots**
![image](https://user-images.githubusercontent.com/16760685/124956067-e70a1780-e017-11eb-9b9f-8d251f878eb2.png)

**Additional context**
Here is a full log for a failed check run producing the error: https://github.com/goatcorp/FFXIVQuickLauncher/runs/3021016089?check_suite_focus=true
My project ID is ``ffxivquicklauncher``.

**Comment by ErikQQY** [comment]
Same issue here🙋‍♂️

The crowdin-cli fails to fetch project info

GitHub action link: https://github.com/casnode/casnode-website/runs/3021183563

**Comment by andrii-bodnar** [comment]
Hi @goaaats @ErikQQY!

Sorry for the inconvenience, we are currently working on the fix and will release it ASAP.

**Comment by ErikQQY** [comment]
@andrii-bodnar Thank you for your prompt response, looking forward to the bug fix.

**Comment by anthonymonori** [comment]
We are also seeing this issue, and we are using an older version of the CLI (3.5.4), which I assume OP does too. 3.6.3 seems to have this property there and shouldn't cause any serialization issue.

**Comment by anthonymonori** [comment]
Not sure if it helps, but we're seeing the same issue bit using the jar directly instead of github actions hence the stacktrace points to `XmlFileImportOptions` rather:

```
❌ Failed to collect project info. Please contact our support team for help
❌ Error from server: <Code: <empty_code>, Message: Unrecognized field "customSegmentation" (class com.crowdin.client.sourcefiles.model.XmlFileImportOptions), not marked as ignorable (4 known properties: "translatableElements", "translateContent", "translateAttributes", "contentSegmentation"])
 at [Source: (String)"{"translateContent":true,"translateAttributes":false,"contentSegmentation":true,"translatableElements":[],"customSegmentation":false}"; line: 1, column: 133] (through reference chain: com.crowdin.client.sourcefiles.model.FileInfoResponseList["data"]->java.util.ArrayList[37]->com.crowdin.client.sourcefiles.model.FileResponseObject["data"]->com.crowdin.client.sourcefiles.model.File["importOptions"]->com.crowdin.client.sourcefiles.model.XmlFileImportOptions["customSegmentation"])>
```

We are also using v3.5.4, and I can see this issue is non existent on 3.6.3 (`XmlFileImportOptions` does contain the new `customSegmentation` property).

Has the lowest supported version of the cli been increased?

**Comment by andrii-bodnar** [comment]
This issue (`XmlFileImportOptions` serialization) related to the older versions. Updating to the CLI **3.6.3** or **3.6.4** should fix this.

Issue related to the `OtherFilesImportOption` serialization is fixed in CLI **3.6.4** and this version is already available. Updating to this version should fix this issue.

Crowdin GitHub action will be updated soon.

**Comment by andrii-bodnar** [comment]
We faced some issues with docker image deploying, so the GitHub Action new version release a bit delaying. Sorry again for the inconvenience, will keep you posted.

**Comment by andrii-bodnar** [comment]
The new version of Crowdin GitHub Action is released - **1.1.2**

it contains CLI 3.6.4 with the fix.

**Comment by goaaats** [comment]
Thank you very much for the quick fix.
